views/offramp/offramp-residence.pug

```diff
@@ -0,0 +1,20 @@
+extends ../base
+include ../_includes/banner
+
+block variables
+  -var title = __('This service is only available to residents of Ontario')
+
+block content
+
+  h1 #{title}
+  
+  .offramp
+    p #{__('Since you weren’t a resident of Ontario on December 31, 2018, you can’t use this service to file your taxes.')}
+
+    div
+      p #{__('There are also other free and secure ways to file your taxes:')}
+      ul
+        li <a href=#{__('https://www.canada.ca/en/revenue-agency/services/tax/individuals/community-volunteer-income-tax-program.html')}>#{__('Visit a free tax clinic near you. ')}</a> #{__('You can have a volunteer help you submit your taxes.')}
+        li #{__('Contact CRA at')} #[a(href="tel:123-456-7890") (123) 456-7890] #{__('for help')}
+        li #{__('File your taxes with')} <a href=#{__('https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/netfile-overview/certified-software-netfile-program.html')}>#{__('other NETFILE software certified by CRA')}</a>
```

Comment:
These bullet points have been repeated several times. Can you put this into an includes maybe?

Also, our current offramp page has a bullet list that looks like this. Can you confirm with Brian whether we can reuse the bullet points we have on the other offramp page?

Reply:
I think the first bullet on the other offramp page would become obsolete with the new offramp pages. If we could delete it, all bullets would be consistent, I'll double check with Brian